### PR TITLE
Refactor DaPars to include mandatory run modes

### LINKED ISSUE #204: Refactor DaPars execution workflow to include run modes in the config file
DaPars qualifies for identification and differential challenges. Right now, it only has a run_differential boolean in its modules.config file to indicate whether to run identification when set to false, or run differential when set to true. 

Accroding to issue #189, we would like to have run_identification and run_differential booleans to obtain any combination of output files. This allows for more flexibility for the user.

### PR DESCRIPTION
**Pull request by faricazjj**
Fixes #204 

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated corresponding READMEs (if applicable)
- [x] My code follows the templates/style guidelines of the repository
- [x] In- and output formats comply with APAeval specifications
- [x] No parameters or file names are hardcoded
- [x] Results, logs or other output is not commited to the repository
- [x] I have tested my code

### CODE REVIEW

**Review by dominikburri — COMMENTED** [review]
Thanks for the refactoring! 
I tested the execution with all 4 combinations of run modes with `samplesheet_example_files.csv` and Singularity, and each works just fine. I only have a minor comment regarding the readme.

So this part seems to be fine, but maybe it is worth fixing issue #207 with this PR as well. What do you think @faricazjj ?

File: execution_workflows/DaPars/README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ To run with singularity, comment out line 49 in Dapars/nextflow.config file `doc
 ### Parameters
 Parameters used to run the two steps of DaPars are specified in conf/modules.config file. 
 Parameters relevant to the workflow itself are:
+- `run_identification` - set to true to obtain identification challenge output. Specifying any other value will throw an error.
+- `run_differential` - set to true to obtain differential challenge output. Specifying any other value will throw an error.
 - `mode` - whether to run to obtain identification ("identification") or differential ("differential") challenge output.
```
[the remark this answers is not in the document]
Comment (faricazjj) [reply execution_workflows/DaPars/README.md]: yes you're right!

**Comment by faricazjj** [comment]
@dominikburri Thanks so much for reviewing! :D I just addressed your comment and it's now ready for rereview!

As for the bug you reported, I created a separate PR to fix it here https://github.com/iRNA-COSI/APAeval/pull/208. I thought it would be cleaner to link a separate pr to the issue and so that it's easier to review as well   😊

**Review by dominikburri — APPROVED** [review]

**Review by yuukiiwa — APPROVED** [review]
This looks really good, @faricazjj!! :)

**Pull request merged**